bin/build_fixups.sh

```diff
@@ -32,30 +32,32 @@ else
 fi
 # --- end standard header ----
 
+OPENMP_EXTRA_EXAMPLES=$AOMP/share/openmp-extras
```

Comment:
Small request, can this be changed to OPENMP_EXTRAS_EXAMPLES?

Reply:
Variable name changes to OPENMP_EXTRAS_EXAMPLES,  as  suggested